ele/lib/elements.json

```
@@ -3,708 +3,944 @@
     "symbol": "Po",
     "atomic number": 84,
-    "mass": 209.0
+    "mass": 209.0,
+    "radius_bondi": null,
+    "radius_alvarez": null
   },
   "astatine": {
     "name": "astatine",
     "symbol": "At",
     "atomic number": 85,
-    "mass": 210.0
+    "mass": 210.0,
+    "radius_bondi": null,
+    "radius_alvarez": null
   },
   "radon": {
     "name": "radon",
     "symbol": "Rn",
     "atomic number": 86,
-    "mass": 222.0
+    "mass": 222.0,
+    "radius_bondi": null,
+    "radius_alvarez": null
   },
   "francium": {
     "name": "francium",
     "symbol": "Fr",
     "atomic number": 87,
-    "mass": 223.0
+    "mass": 223.0,
+    "radius_bondi": null,
+    "radius_alvarez": null
   },
   "radium": {
     "name": "radium",
     "symbol": "Ra",
     "atomic number": 88,
-    "mass": 226.0
+    "mass": 226.0,
+    "radius_bondi": null,
+    "radius_alvarez": null
   },
   "actinium": {
     "name": "actinium",
     "symbol": "Ac",
     "atomic number": 89,
-    "mass": 227.0
+    "mass": 227.0,
+    "radius_bondi": null,
+    "radius_alvarez": 2.8
   },
   "thorium": {
     "name": "thorium",
     "symbol": "Th",
     "atomic number": 90,
-    "mass": 232.04
+    "mass": 232.04,
+    "radius_bondi": null,
+    "radius_alvarez": 2.93
   },
   "proactinium": {
     "name": "proactinium",
     "symbol": "Pa",
     "atomic number": 91,
-    "mass": 231.04
+    "mass": 231.04,
+    "radius_bondi": null,
+    "radius_alvarez": 2.88
   },
   "uranium": {
     "name": "uranium",
     "symbol": "U",
     "atomic number": 92,
-    "mass": 238.03
+    "mass": 238.03,
+    "radius_bondi": 1.86,
+    "radius_alvarez": 2.71
   },
   "neptunium": {
     "name": "neptunium",
     "symbol": "Np",
     "atomic number": 93,
-    "mass": 237.0
+    "mass": 237.0,
+    "radius_bondi": null,
+    "radius_alvarez": 2.82
   },
   "plutonium": {
     "name": "plutonium",
     "symbol": "Pu",
     "atomic number": 94,
-    "mass": 244.0
+    "mass": 244.0,
+    "radius_bondi": null,
+    "radius_alvarez": 2.81
   },
   "americium": {
     "name": "americium",
     "symbol": "Am",
     "atomic number": 95,
-    "mass": 243.0
+    "mass": 243.0,
+    "radius_bondi": null,
+    "radius_alvarez": 2.83
   },
   "curium": {
     "name": "curium",
     "symbol": "Cm",
     "atomic number": 96,
-    "mass": 247.0
+    "mass": 247.0,
+    "radius_bondi": null,
+    "radius_alvarez": 3.05
   },
   "berkelium": {
     "name": "berkelium",
     "symbol": "Bk",
     "atomic number": 97,
-    "mass": 247.0
+    "mass": 247.0,
+    "radius_bondi": null,
+    "radius_alvarez": 3.4
   },
   "californium": {
     "name": "californium",
     "symbol": "Cf",
     "atomic number": 98,
-    "mass": 251.0
+    "mass": 251.0,
+    "radius_bondi": null,
+    "radius_alvarez": 3.05
   },
   "einsteinium": {
     "name": "einsteinium",
     "symbol": "Es",
     "atomic number": 99,
-    "mass": 252.0
+    "mass": 252.0,
+    "radius_bondi": null,
+    "radius_alvarez": 2.70
```

precision only goes to 2.7
```suggestion
    "radius_alvarez": 2.7
```